test/unit/analysis/mixed_temporal_merge_test.py

```diff
@@ -0,0 +1,358 @@
+"""Tests for merging datasets at different temporal frequencies."""
+import pandas as pd
+from pandas.tseries.offsets import BYearEnd
+
+import pudl.helpers
+
```

Comment:
The `quarter` case is a little unusual, not being used currently, and also not being tested. It seems like we should either test it, or remove it (and add it in when we need to for the FERC data).

Reply:
I put in a place holder test for merging quarterly data onto monthly data but this should be updated to use FERC data for testing when we need to do this merge for the FERC data.